packages/graphql/src/schema/scalars/BigInt.ts

```diff
@@ -17,32 +17,44 @@
  * limitations under the License.
  */
 
-import { GraphQLScalarType, Kind, ValueNode } from "graphql";
-import { int, Integer } from "neo4j-driver";
+import { GraphQLError, GraphQLScalarType, Kind, ValueNode } from "graphql";
+import { int, isInt } from "neo4j-driver";
 
 export default new GraphQLScalarType({
     name: "BigInt",
     description:
         "A BigInt value up to 64 bits in size, which can be a number or a string if used inline, or a string only if used as a variable. Always returned as a string.",
-    serialize(value: Integer) {
-        return value.toString(10);
+    serialize(outputValue: any) {
```

Comment:
We may want to consider start using `unknown` here and in `parseValue`

```suggestion
    serialize(outputValue: unknown) {
```